genet/utils/graph_operations.py

```diff
@@ -270,6 +272,18 @@ def build_attribute_dataframe(iterator, keys: Union[list, str], index_name: str
     return df
 
 
+def get_pandas_dtype(dict):
```

Comment:
As discussed - this feels pretty hacky, but it gets the job done and only broke a single test in the end. I'm still hoping for a more elegant solution, but at least we know this does work.

Reply:
I think it's fine. I wonder where we should put this method, there is another one here: https://github.com/arup-group/genet/blob/master/genet/utils/dict_support.py#L115 which is also in theme of working with pandas data, and `dict_support` is not really the best place for it... should we maybe start a new module in utils.. ?

Reply:
Yes, I also wondered if this was the best place for it. Perhaps a new `pandas-utils` module in utils is the thing?